lib/views/home_content/map/map_screen.dart

```diff
@@ -0,0 +1,50 @@
+import "package:flutter/material.dart";
+import "package:flutter_hooks/flutter_hooks.dart";
+import "package:hooks_riverpod/hooks_riverpod.dart";
+import "package:proxima/viewmodels/map/map_view_model.dart";
+import "package:proxima/views/home_content/map/post_map.dart";
+import "package:proxima/views/sort_option_widgets/feed_sort_option/map_selection_option_chips.dart";
+
+class MapScreen extends HookConsumerWidget {
+  const MapScreen({super.key});
+
+  static const mapScreenKey = Key("map");
+  static const dividerKey = Key("Divider");
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final mapState = ref.watch(mapNotifierProvider);
+    final mapNotifier = ref.watch(mapNotifierProvider.notifier);
+
+    useEffect(
```

Comment:
This code really does not feel right to me. There are a lot of side effects + combinations of two "hacky" ways of doing things: `useEffect` and `Future.microtask`... It should be refactored and dealt with before merging.

Reply:
I removed this part of the code and made it simpler.